test/workers/find_and_delete_accounts_scheduled_worker_test.rb

```diff
@@ -9,10 +9,6 @@ def setup
   end
 
   def test_perform
-    ThreeScale.config.stubs(onpremises: false)
-    DeleteAccountHierarchyWorker.expects(:perform_later).never
-    FindAndDeleteScheduledAccountsWorker.new.perform
-
     ThreeScale.config.stubs(onpremises: true)
```

Comment:
Imo you should remove this line too, because we want it to behave this way no matter if it is on saas or on prem 😄